src/marks/text.js

```diff
@@ -114,12 +127,14 @@ export class Text extends Mark {
   }
 }
 
-function applyMultilineText(selection, {monospace, lineAnchor, lineHeight, lineWidth}, T) {
+function applyMultilineText(selection, {monospace, lineAnchor, lineHeight, lineWidth, textOverflow}, T, TL) {
   if (!T) return;
+  if (textOverflow && !isFinite(lineWidth)) throw new Error(`the textOverflow option requires a specified lineWidth`);
+  const measure = monospace ? monospaceWidth : defaultWidth;
   const linesof = isFinite(lineWidth)
-    ? monospace
-      ? (t) => lineWrap(t, lineWidth, monospaceWidth)
-      : (t) => lineWrap(t, lineWidth * 100, defaultWidth)
+    ? textOverflow
+      ? (t) => [overflow(t, lineWidth * 100, measure, textOverflow)]
+      : (t) => lineWrap(t, lineWidth * 100, measure)
```

Comment:
I’d like to investigate promoting this into separate functions (similar to what we do for facetAnchor) so there is less case branching during evaluation.